src/ocean/math/Distribution.d

```diff
@@ -165,7 +165,7 @@ public class Distribution ( T )
 
     ***************************************************************************/
 
-    public void opCatAssign ( T value )
+    public void opOpAssign ( string op ) ( T value ) if (op == "~")
```

Comment:
This actually covered by the test cases using https://github.com/sociomantic-tsunami/ocean/blob/v5.x.x/src/ocean/math/Distribution.d#L453

Reply:
This change will need to update documentation parameters